### ISSUE TITLE
python wheel package is "empty"

### ISSUE THREAD

**Issue by RonnyPfannschmidt** [NixOS/nixpkgs#16243]
## Issue description
```
$ tree /nix/store/lsg86rphp6lcadijmnyfbsl3dqy2ni2d-python2.7-wheel-0.29.0
/nix/store/lsg86rphp6lcadijmnyfbsl3dqy2ni2d-python2.7-wheel-0.29.0
├── lib
│   └── python2.7
│       └── site-packages
└── nix-support
    └── propagated-native-build-inputs
```

### Steps to reproduce

install wheel from pythonpackages in order to get setup.py bdist_wheel

## Technical details
```
$ nixos-version
16.09.git.47e8957 (Flounder)
$ nix-env --version
nix-env (Nix) 1.11.2
$ nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion
"16.03.923.9e959ce"

```

im running a nixos from a git checkout due to testing bug-fixes on gnome
im running a nix-env from the stable channel to work on my projects

**Comment by domenkozar** [comment]
This is because wheel package is installed using bootstrapped-pip, which already contains wheel package so it will skip the installation.

**Comment by RonnyPfannschmidt** [comment]
so the actual wheel package is transiently broken? how do i properly put a wheel into my system then

**Comment by domenkozar** [comment]
Workaround is to depend on `bootstrapped-pip` as it provides wheel package

**Comment by RonnyPfannschmidt** [comment]
where in the tree is i tried the following which doesnt have that as attribute
```nix
with import <nixpkgs> {};
with pkgs;
with python27Packages;
stdenv.mkDerivation rec {
  name = "sentaku-test";
  src = null;
  buildInputs = [
    gitFull
    less
    ncurses
    openssh
    bpython
    sqlite
    setuptools_scm
    bootstrapped-pip
    pip
    wheel
    pytest
    flake8
    pygments
    sphinx
    requests
    selenium
    virtualenv
    watchdog
  ];

  shellHook=''
  export PS1="(${name}) \w \\$ \[$(tput sgr0)\]"
  export PYTHONPATH="$PYTHONPATH:./src"
  test() {
    py.test
  }

  build_docs() {
    sphinx-build -b html -d build/sphinx-doctrees docs build/htmldocs
  }

  autobuild_docs() {
    watchmedo shell-command docs src -R \
      -c 'sphinx-build -b html -d build/sphinx-doctrees docs build/htmldocs'
  }

  clean() {
    find -name \*.pyc -delete
    find -name __pycache__ -delete
    rm -rf build/sphinx-doctrees build/htmldocs
  }

  build_fresh_docs() {
    clean
    build_docs
  }

  test_clean() {
    clean
    test
  }

  '';
}
```

**Comment by domenkozar** [comment]
See https://github.com/NixOS/nixpkgs/pull/16245

**Comment by RonnyPfannschmidt** [comment]
its not clear to me how to apply that to a shell env, do i need to use override until the patch lands?

**Comment by FRidh** [comment]
fixed in 9f2699fd303095ca76724c804b0d91c662da11b0
